docs/modernization/InkToolbar.md

```diff
@@ -0,0 +1,94 @@
+---
+title: InkToolbar control for Windows Forms and WPF
+author: granitestatehacker
+description: This control is a wrapper to enable use of the UWP InkToolbar control in Windows Forms or WPF.
+keywords: windows 10, uwp, windows community toolkit, uwp community toolkit, uwp toolkit, InkToolbar, Windows Forms, WPF
+---
+
+# InkToolbar controls for Windows Forms and WPF applications
+
+The **InkToolbar** controls provide a common interface to manage an InkCanvas for Ink-based user interaction in your Windows Forms or WPF desktop application.
+
+![Web View Samples](../resources/images/Controls/InkCanvas.png)
+
+These controls use the Windows 10 implementation, and are used to embed a panel that can be drawn on, in Ink-style user interaction.  
+
+## About InkToolbar controls
+
+The Windows Forms version of this control is coming soon. It will be located in the **Microsoft.Toolkit.Win32.UI.Controls.WinForms** namespace. 
+
+The WPF version is located in the **Microsoft.Toolkit.Win32.UI.Controls.WPF** namespace. 
```

Comment:
Please check all the namespaces, package names, and file locations in this document